### PR TITLE
add details about myself(Prabin)

### PR DESCRIPTION
**Pull request by PrabinSingh**
Maile aafno barema details add garya ho.

### CODE REVIEW

**Comment by MegaMind98** [comment]
Merged, @PrabinSingh !

File: _class/prabin_singh.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+firstname: Prabin
+lastname: Singh
+photo: prabin_singh.jpg
+place: Balkumari
+email: sprabin006@gmail.com
+phonenumber: 9812083274
+facebook: https://www.facebook.com/prabin.singh.14473
+twitter:
+linkedin:
```
Comment (PrabinSingh) [inline _class/prabin_singh.md:10]: https://www.linkedin.com/in/prabin-singh-a3b11b139/

File: _class/prabin_singh.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+firstname: Prabin
+lastname: Singh
+photo: prabin_singh.jpg
+place: Balkumari
+email: sprabin006@gmail.com
+phonenumber: 9812083274
+facebook: https://www.facebook.com/prabin.singh.14473
+twitter:
```
Comment (PrabinSingh) [inline _class/prabin_singh.md:9]: https://twitter.com/Prabinsingh054

**Comment by MegaMind98** [comment]
What did you intend to change, Prabin?

**Pull request merged**